.gitignore

```diff
@@ -10,3 +10,5 @@
 
 # Output of the go coverage tool, specifically when used with LiteIDE
 *.out
+*.iml
```

Comment:
add `.idea` to ignore, but before remove all `.idea` subdirectories and files from git.